CHANGELOG.md

```diff
@@ -2,11 +2,29 @@
 All notable changes to [Solang](https://github.com/hyperledger/solang/)
 will be documented here.
 
-## Unreleased
+## v0.3.2 Brasília
+
+The language server is much improved, and many fixes all over.
+
+### Added
+- Go to definition is now implemented in the language server. [chioni16](https://github.com/chioni16)
+- The parser has been updated to be compatible with Ethereum Solidity v0.8.21. [seanyoung](https://github.com/seanyoung)
 
 ### Fixed
 - **breaking** Resolving import paths now matches solc more closely, and only resolves relative
   paths when specified as `./foo` or `../foo`. [seanyoung](https://github.com/seanyoung)
+- **Solana** The `lamports` and `data` fields of `tx.accounts` can be modified again. [LucasSte](https://github.com/LucasSte)
+- It is not longer necessary to save a Solidity file, in order for the language server to pick
+  up changes to the file. [chioni16](https://github.com/chioni16)
+- The negate operator `-` now checks for overflow at runtime, and other math overflow fixes.
+  [seanyoung](https://github.com/seanyoung)
+
+### Changed
+- The Substrate target has been renamed to Polkadot. [xermicus](https://github.com/xermicus)
+- **Polkadot** `assert()` and `require()` is now implemented as a transction revert, rather
+  than a trap. The error data is returned, and encoded the same as on Ethereum. Error data is now
+  passed to the calling contract, all the way up the call stack. [xermicus](https://github.com/xermicus)
+- **Polkadot** constructor can be non-payable. [xermicus](https://github.com/xermicus)
```

Comment:
```suggestion
- **Polkadot** constructor can be non-payable. [xermicus](https://github.com/xermicus)
- The semantic analyzer always writes storage variables, regardless whether they are read or not. [xermicus](https://github.com/xermicus)
```

Reply:
Feel free to edit my suggestion with a version you think it is more accurate

Reply:
Looks good, but "no longer removed" would confuse me. What does it mean for the compiler to "remove" storage variables? What happens to them?

What happens is that writes to unused storage variables were eliminated and this changed. What about a combination of both: 

`Storage variables are now always written to, regardless whether the contract does read them or not. Prior behavior was to not write to storage variables if they are not read, which can remove wanted side effects, because unused storage variables may be used in future versions of the contract.`

Reply:
I agree that is much clearer, I've updated it with this text